docs/authentication/configuration/session-options.mdx

```diff
@@ -42,6 +42,9 @@ By default, this setting is enabled with a default value of 7 days for all newly
 1. Toggle on **Maximum lifetime**.
 1. Set your desired duration.
 
+> [!NOTE]
+> Unfortunately, Safari will clear Clerk's session cookie every 7 days as an effect of their [CNAME cloaking ITP policy](https://webkit.org/blog/11338/cname-cloaking-and-bounce-tracking-defense/). This will cause users to be signed out weekly, even if session lifetime is set to a longer duration. The only current workaround to this issue is to [proxy FAPI](/docs/advanced-usage/using-proxies).
```

Comment:
```suggestion
> Safari will clear Clerk's session cookie every 7 days as an effect of their [CNAME cloaking ITP policy](https://webkit.org/blog/11338/cname-cloaking-and-bounce-tracking-defense/). This will cause users to be signed out weekly, even if session lifetime is set to a longer duration. The only current workaround to this issue is to [proxy FAPI](/docs/advanced-usage/using-proxies).
```